Modules/feed/feed_controller.php

```diff
@@ -55,7 +55,7 @@ function feed_controller()
                 if ($feed->exist($feedid)) {  // if the feed exists
                    $f = $feed->get($feedid);
                    if ($f['public'] || ($session['userid']>0 && $f['userid']==$session['userid'] && $session['read'])) {
-                       $result[$i] = 1*$feed->get_value($feedid); // null is a valid response
+                       $result[$i] = $feed->get_value($feedid); // null is a valid response
```

Comment:
The answer should return numeric values and "null".
Multiply by 0 will make null return as 0 that is wrong.